### PR TITLE
draft GO

### PR DESCRIPTION
**Pull request by T3C42**

### CODE REVIEW

File: content/geschaftsordnung.md [hunk 1]
```diff
@@ -11,122 +11,108 @@ tocopen: true
...
 
-## 9 Beschlussfähigkeit
+## § 9 Beschlussfähigkeit
 Das Festessen ist beschlussfähig, wenn ordnungsgemäß eingeladen wurde.
 Zusätzlich ist das Vorhandensein oder das glaubhaft in Aussicht gestellte Vorhandensein von
 Brokkoli oder eines Brokkoli-Äquivalents während des Festessens notwendig. Ein Brokkoli-
 Äquivalent muss mit Zwei-Drittel-Mehrheit als solches akzeptiert werden. Das Vorhandensein
 einschließlich der Akzeptanz darf zu einem beliebigen Zeitpunkt des Festessens erfolgen und
 somit auch das bisherige Festessen legitimieren. Ausgeschlossen von dieser Äquivalenz-
 Akzeptanz sind Objekte nach Anlage I-III zu ##1 Absatz 1 des BtMG. Konsum dieser führt zu
```
Comment (T3C42) [inline content/geschaftsordnung.md:57]: up for discussion

File: content/geschaftsordnung.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: "Geschäftsordnung"
 summary: "asdf"
 date: 2023-05-03T10:15:07+02:00
-draft: false
+draft: true
```
Comment (isiko) [inline content/geschaftsordnung.md]: Is there a reason for this? Setting this to true would result in this not being published on the Website

File: content/geschaftsordnung.md (hunk 1 above)
Comment (isiko) [reply content/geschaftsordnung.md:57]: Either would be fine for me, your version is probably better

**Comment by isiko** [comment]
Moving this to "Ready for Review" as a review was requested :upside_down_face:

**Review by isiko — CHANGES_REQUESTED** [review]

File: content/geschaftsordnung.md (hunk 1 above)
Comment (isiko) [reply content/geschaftsordnung.md:57]: Either would be fine for me, your version is probably better

Comment (T3C42) [reply content/geschaftsordnung.md:57]: right now there is only one option presented (line 56 f)

**Review by isiko — APPROVED** [review]

File: content/geschaftsordnung.md (hunk 2 above)
Comment (T3C42) [reply content/geschaftsordnung.md]: resolved in "undo draft"

**Pull request merged**